doc/cabal-package-description-file.rst

```diff
@@ -2676,43 +2676,31 @@ Starting with Cabal-2.2 it's possible to use common build info stanzas.
 
     TBW
 
-Source Repositories
-^^^^^^^^^^^^^^^^^^^
+
+.. _pkg-author-source:
+
+Package Source
+^^^^^^^^^^^^^^
 
 .. pkg-section:: source-repository
     :since: 1.6
 
-It is often useful to be able to specify a source revision control
-repository for a package. Cabal lets you specify this information in
-a relatively structured form which enables other tools to interpret and
-make effective use of the information. For example the information
-should be sufficient for an automatic tool to checkout the sources.
-
-Cabal supports specifying different information for various common
-source control systems. Obviously not all automated tools will support
-all source control systems.
+Where to find the source for this package.
```

Comment:
better "source code"

Reply:
Expanded it to "Marks out where to find the *source code* for this package within a **source code repository**."